resource-server.tf

```diff
@@ -0,0 +1,16 @@
+resource "aws_cognito_resource_server" "resource_server" {
+  for_each = { for resource in var.resource_servers : resource.identifier => resource }
```

Comment:
We need to force an empty list if `module_enabled = false`

Reply:
good point! Thanks!